configurecompiler.cmake

```diff
@@ -341,8 +341,7 @@ elseif (CLR_CMAKE_PLATFORM_UNIX)
   # This linker option causes executables we build to be marked as containing
   # position independent code.
   # It is necessary to make ASLR work for executables.
-  set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--build-id=sha1 -Wl,-z,relro,-z,now")
-  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pie -Wl,-z,relro,-z,now")
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pie")
```

Comment:
This removes the `-Wl,-z,relro,-z,now` but it is needed for Linux. Can you please add it to the  CMAKE_SHARED_LINKER_FLAGS and CMAKE_EXE_LINKER_FLAGS in the Linux if block below?

Reply:
Yes. Do we need it on FreeBSD as well?

Reply:
No, just Linux (looking at the flags set in main)

Reply:
Thanks.